### ISSUE TITLE
Getting error "Metric name is None! Skipping metric value retrieval"

### ISSUE THREAD

**Issue by abhijeetdhakane** [ashleve/lightning-hydra-template#459]
I was using custom loss function built using `torch` operations. I was getting error/warning every time run the program
```
[src.utils.utils][INFO] - Metric name is None! Skipping metric value retrieval...
```

**Comment by ashleve** [comment]
It's not an error or warning - it's just an informational console log.

Metric name is only needed when doing hyperparameter search so hydra knows which metric to optimize.

It can be set here:
https://github.com/ashleve/lightning-hydra-template/blob/cd640a0153c117934ad95f55da2e1aa2c936d00f/configs/hparams_search/mnist_optuna.yaml#L9-L11

Closing for now, feel free to reopen if you have any questions!
